smartmin/users/views.py

```diff
@@ -274,6 +275,22 @@ class Forget(SmartFormView):
         success_url = "@users.user_login"
         fields = ('email', )
 
+        def build_email_context(self):
+            context = Context()
+
+            processors = []
+            collect = []
+            collect.extend(('smartmin.users.context_processors.links_components',))
```

Comment:
This is slightly weird. I like the default for EMAIL_CONTEXT_PROCESSORS being this, but we shouldn't prepend it if the user sets it to something else.